samples/ios/Continote/Continote/MyNotesViewController.swift

```diff
@@ -15,39 +15,155 @@
 //
 
 import UIKit
+import Firebase
+import FirebaseDatabaseUI
+import MaterialComponents.MaterialButtons
 
 /**
- This view controller presents the user with a list of their notes in Continote.
- From here the user can open or delete notes.
+ The view controller that presents the user with a list of their Notes in Continote.
+
+ From here the user can add, delete, or open to edit Notes.
  */
 class MyNotesViewController: BaseViewController {
 
+  // Firebase Realtime Database reference for the current user's Notes within Continote.
+  private var notesFirebaseDatabaseRef: DatabaseReference?
+
+  // The data source to populate the TableView of Notes for the current user.
+  private var myNotesTableViewDataSource: MyNotesTableViewDataSource?
+
   // UI outlets
-  @IBOutlet var temporaryTodoLabel: UILabel!
+  @IBOutlet var myNotesTableView: UITableView!
```

Comment:
`tableView` would probably be fine here, unless there's a second tableView in this.